### PR TITLE
Here's what I have at the moment. Still isn't correctly comparing the…

### PR DESCRIPTION
**Pull request by joshuayoung101**
… entered data with that in the file, but edit and reading data from the file and the menu to select init and edit are working.

### CODE REVIEW

File: Assignment2/main.cpp [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        do {
+            std::cout << "Enter your password" << std::endl;
+            getline(std::cin, Pass);
+        }while (Pass != saved_Pass);
+
+        std::string Timezone = "";
+        do {
+            std::cout << "Enter your timezone" << std::endl;
+            getline(std::cin, Timezone);
+        } while (Timezone != saved_Timezone);
```
Comment (ninjapanzer) [inline Assignment2/main.cpp]: You should have be setting the knownrecipients location here too within this init section

File: Assignment2/main.cpp [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+
+
+int main() {
+
+    std::string menu = "";
+    std::cout << "Enter init or edit" << std::endl;
+    std::cin >> menu;
+    std::cout << "Menu selected" << std::endl;
+    if (menu == "edit") {
```
Comment (ninjapanzer) [inline Assignment2/main.cpp]: This section is a good start! Here is what you need to consider to complete it. How can you know what is in the file so you only have to ask the user to supply one input for the property/field named by `argv[2]`

the invocation would be `gen.exe edit email` for example. So the program will only ask for the email input but will have to maintain the other values in the file.
HINT if I just told you to recreate the file from the contents of the file. Like copying the file what could you do procedurally to make that happen.

File: Assignment2/main.cpp [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        do {
+            std::cout << "Enter your name(first and last)" << std::endl;
+            getline(std::cin, Username);
+        }while (Username != saved_Username);
+
+        std::string Email = "";
+        do {
+            std::cout << "Enter your Email address" << std::endl;
+            getline(std::cin, Email);
+        }while (Email != saved_Email);
```
Comment (sshurie) [inline Assignment2/main.cpp]: Since all of your saved_ variables store the same value, I think you can simplify this by writing just one variable storing the empty quotation marks. For example, " std::string empty = "" " and then all of your do loops could be something like " do{...}while(Email != empty)."

Alternatively, there is also another method that can be used to test to make sure that the length of the user input variable is not equal to zero, and with this we could completely remove the empty saved_ string variables.

Good job so far, this is definitely a hard assignment for all of us! I remember you asked in class on Wednesday about using arrays to write the output into the files. If you would like to see that in action, I included a few arrays in my assignment to help with file handling. Feel free to check it out! :)
